src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -28,8 +28,11 @@ export default function FormNav(props) {
   const formPages = createFormPageList(formConfig);
   const pageList = createPageList(formConfig, formPages);
 
-  const eligiblePageList = getActiveExpandedPages(pageList, formData);
-
+  const eligiblePageList = getActiveExpandedPages(
+    pageList,
+    formData,
+    isLoggedIn,
```

Comment:
It'd be nice to pass an object as the third parameter... personally, I'd like to see `featureToggles` also passed to the depends function, so it could be done in this work, or in the future without adding a whole bunch of parameters